packages/blog/data/_posts/2020/2021-01-18-smooth-upgrade-to-nette-31-in-diffs.md

```diff
@@ -0,0 +1,239 @@
+---
+id: 14
+title: "Smooth Upgrade to Nette 3.1 in Diffs"
+perex: |
+    Nette 3.1 was released almost a month ago. Packages using it had enough time to give support to small BC breaks and now it's ready to run on your project.
+    Let's look at what has changed and how to upgrade today.
+
+contributor: "lulco"
+pull_request_id: 5195
```

Comment:
@lulco This now allows cool contributor badges like these :) 

![image](https://user-images.githubusercontent.com/924196/105772801-0aa63d80-5f63-11eb-9306-72159569a165.png)

Reply:
Cool :)